src/net/tcp.rs

```diff
@@ -64,28 +64,28 @@ impl TcpListener {
                 match pending.poll().expect("shouldn't be canceled") {
                     Async::NotReady => {
                         self.pending_accept = Some(pending);
-                        return Err(io::ErrorKind::WouldBlock.into())
+                        return Err(io::ErrorKind::WouldBlock.into());
```

Comment:
In general could changes like this w/ rustfmt be held off for now? It tends to be pretty burdensome to be in a "half formatted" state and it feels generally best to wait until rustfmt is ready to go and then we just turn it on all at once.